examples/regexp-find-operations/regexp_find_operations.bal

```diff
@@ -0,0 +1,55 @@
+import ballerina/io;
+import ballerina/lang.regexp;
+
+public function main() {
+    string logContent = string `
```

Comment:
imo, its better to include non matching example as well

Reply:
There are other non-matching info and warn logs in this logContent